Products/CMFPlone/RegistrationTool.py

```diff
@@ -422,9 +422,10 @@ def registeredNotify(self, new_member_id):
         # don't need to worry about 'UseMailHost' permissions).
         registry = getUtility(IRegistry)
         encoding = registry.get('plone.email_charset', 'utf-8')
-        mail_text = self.registered_notify_template(
-            self, self.REQUEST, member=member, reset=reset, email=email,
-            charset=encoding)
+        registered_notify_template = getMultiAdapter(
```

Comment:
On the first glance this sounds tempting but getting the multi-adapter is well documented and changing it would lead to different patterns all over the place.